slidetoact/src/main/res/values/attrs.xml

```diff
@@ -8,6 +8,8 @@
             <enum name="bold" value="1" />
             <enum name="italic" value="2" />
         </attr>
+        <attr name="text_all_caps" format="boolean" />
```

Comment:
So I think that maybe having those `text_` attributes was a bad API decision.
I'd rather got for having a:
```suggestion
        <attr name="android:textAppearance" />
```
And allow customization on the style level.
We'll keep on having the other `text_` attributes for backward compatibility.